src/bot/commands/user_commands.py

```diff
@@ -137,7 +137,7 @@ def run_get_player_hero_winrate_command(update, context):
     if status_code != constants.HTTP_STATUS_CODES.OK.value:
         update.message.reply_text(constants.BAD_RESPONSE_MESSAGE)
 
-    hero_data = helpers.filter_hero_winrates(response, str(hero["id"]))
+    hero_data = helpers.filter_hero_winrates(response, str(hero_id))
```

Comment:
Because of the dynamic nature of python, the conversion to string should be handled by the internal function in this case. Looking at the filter_hero_winrates function, I don't know if it should be a string until I look at the code that calls it. Everywhere else in the codebase, a hero_id is treated as an integer and we should pass it as such.